src/radical/pilot/pmgr/launching/default.py

```diff
@@ -1269,8 +1268,8 @@ def _stage_out(self, pilot, sds):
             sd['prof_id'] = pilot['uid']
 
         for sd in sds:
-            sd['source'] = str(complete_url(sd['source'], self._rem_ctx, self._log))
-            sd['target'] = str(complete_url(sd['target'], self._loc_ctx, self._log))
+            sd['source'] = str(complete_url(sd['source'], rem_ctx, self._log))
+            sd['target'] = str(complete_url(sd['target'], loc_ctx, self._log))
```

Comment:
may be I'm missing something, just what I see: `source` uses info from `rem_ctx`, which has `'pwd'     : client_sandbox`, and `target` uses info from `loc_ctx`, which has `'pwd'     : pilot_sandbox`. Thus it is a transfer from client to pilot, which looks like stage in, right? but should be switched for stage out?

Reply:
Yeah, that sounds wrong, thanks!

Reply:
This should be fixed now - would you care to re-review, please @mtitov ?  Thanks!